### PR TITLE
fix: enabled --create-sbom on each build we have

### PR DESCRIPTION
**Pull request by zdtsw**
should have merged before https://github.com/adoptium/temurin-build/pull/3053

Ref: https://github.com/adoptium/temurin-build/pull/3053#issuecomment-1188914473

### CODE REVIEW

**Comment by karianna** [comment]
Couple of quick questions:

1. Were the SBOM capabilities back-ported to Java 11?  Not sure we can SBOM there?
2. Is SBOM == Reproducible Build or just the creation of the SBOM?
3. Will this run on nightlies or only releases?

**Comment by sxa** [comment]
> Couple of quick questions:
> 
>     1. Were the SBOM capabilities back-ported to Java 11?  Not sure we can SBOM there?
>     2. Is SBOM == Reproducible Build or just the creation of the SBOM?
>     3. Will this run on nightlies or only releases?

TL;DR 1a) Yes, 2) No, 3) Both

SBOM creation and reproducible build enablement are separate actions - reproducible is enabled when a RELEASE build is done.
We have SBOM creation for all releases - they are included in the artifacts for all nightly builds as well as releases.

**Review by andrew-m-leonard — APPROVED** [review]

File: pipelines/jobs/configurations/jdk11u_pipeline_config.groovy [hunk 1]
```diff
@@ -174,6 +204,9 @@ class Config11 {
                 test                : [
                         nightly: ['sanity.openjdk'],
                         weekly : ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf']
+                ],
+                buildArgs           : [
```
Comment (sxa) [inline pipelines/jobs/configurations/jdk11u_pipeline_config.groovy:208]: Ther is already a buildArgs section for other variants in this section - please unify them, otherwise it will lead to confusion at best, and at worst not actually work.

**Review by sxa — CHANGES_REQUESTED** [review]
On change required to avoid duplication of a section

**Review by karianna — APPROVED** [review]
LGTM

**Comment by karianna** [comment]
@sxa I think you can approve and merge this tomorrow

File: pipelines/jobs/configurations/jdk17u_pipeline_config.groovy [hunk 2]
```diff
@@ -74,14 +74,14 @@ class Config17 {
...
                 crossCompile        : 'x64',
-                buildArgs           : '--cross-compile',
                 additionalNodeLabels: 'win2016&&vs2019',
                 test                : [
                         nightly: [],
                         weekly : []
                 ],
-                buildArgs           : [
-                        "temurin"   : '--create-jre-image --create-sbom'
+                buildArgs       : [
```
Comment (sxa) [inline pipelines/jobs/configurations/jdk17u_pipeline_config.groovy:82]: OpenJ9 doesn't build on Windows/aarch64 so that doesn't need to be included and may cause confusion, although equally doesn't cause any harm

**Review by sxa — APPROVED** [review]
LGTM now although I suggest we do not merge until the new JDK11 and potentially JDK17 patches are built and shipped (perhaps a further opportunity to remove the cross-compile option listed as being for OpenJ9 on windows/aarch64)

File: pipelines/jobs/configurations/jdk17u_pipeline_config.groovy (hunk 2 above)
Comment (zdtsw) [reply pipelines/jobs/configurations/jdk17u_pipeline_config.groovy:82]: i can remove openj9 ,  no confusion for future change

**Comment by zdtsw** [comment]
@sxa guess we have to merge this now, because https://github.com/adoptium/temurin-build/pull/3053 has been merged ?

**Comment by sxa** [comment]
> @sxa guess we have to merge this now, because [adoptium/temurin-build#3053](https://github.com/adoptium/temurin-build/pull/3053) has been merged ?

ARGH! Yes ... Since @andrew-m-leonard is aware he can keep an eye on any side-effects from merging it
